src/hmem.c

```diff
@@ -168,6 +168,16 @@ struct ofi_hmem_ops hmem_ops[] = {
 	},
 };
 
+bool ofi_hmem_any_ipc_enabled(void)
+{
+	int iface;
+
+	for (iface = 0; iface < ARRAY_SIZE(hmem_ops); iface++)
+		if (ofi_hmem_is_ipc_enabled(iface))
```

Comment:
We probably need to add ```ofi_hmem_is_initialized(iface && ``` to this check.  I missed this in my prior comment.

Reply:
Hmmm... I guess my local non-gpu dev instance wasn't going in there b/c I didn't compile in support for those things, but we can't assume package managers will be the same.  Good Catch.